priya/javascript/Assignment11/JsAssign11Q1.html

```diff
@@ -0,0 +1,66 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+      function show_roll(sorted_mark) {
+        debugger;
+        var p;
+        p = parseInt(document.getElementById("pic").value);
+        let marksRollNo = document.getElementById("demo");
+        let isPresent = false;
+        for (let index = 0; index < sorted_mark.length; index++) {
+          if (sorted_mark[index]["roll_no"] == p) {
+            marksRollNo.innerHTML = `Roll No ${p} has scored ${sorted_mark[index]["marks"]} marks`;
+            isPresent = true;
+            break;
+          }
+        }
+        if (!isPresent) {
+          marksRollNo.innerHTML = `Given Roll No ${p} is Not Present`;
+        }
+      }
+    </script>
+    <title>Roll number and Marks</title>
+  </head>
+  <body>
+    Enter number : <input type="text" id="pic" />
+    <button onclick="show_roll(sorted_mark)">show</button>
```

Comment:
Now Change your button same as below.
```suggestion
    <button onclick="show_roll()">show</button>
```